src/extra/python/isca/templates/compile.sh

```diff
@@ -1,16 +1,15 @@
 #!/usr/bin/env bash
 # Compiles the MiMa GFDL Moist Model (tested on emps-gv2.ex.ac.uk)
 
-
 # 1. Configuration
 hostname=`hostname`
-template={{ template_dir }}/mkmf.template.ia64
+template={{ template_dir }}/mkmf.cheyenne.intel
```

Comment:
This `compile.sh` file is used for all the different architectures so we must not hard code cheyenne or any other specific info in here.